### ISSUE TITLE
Send Point Selector Coordinates to Visualisation

### ISSUE THREAD

**Issue by fayeli** [fayeli/biojs-vis-bonestagram#18]
In the web App, after the user is completed with selecting the points on their image, we need to send the coordinates and image to the visualisation.
- A "Complete" Button
- Sends user back to the top of the page scroll 
- Restart visualisation with new image and coordinates

**Comment by fayeli** [comment]
Just have the most basics of these done!

Some nice-to-haves to be added:
- UI of the "complete/next" buttton. position? on hover animation? icon? text?
- hide "complete/next" buttton initially, show only after user uploaded picture
- message shown e.g. success! your bonestagram is ready!
